composer.json

```diff
@@ -39,8 +39,8 @@
     "contao/manager-bundle": "~4.4"
   },
   "replace": {
-    "contao-legacy/dependency-container": "self.version",
-    "bit3/contao-dependency-container": "self.version"
+    "contao-legacy/dependency-container": "<1.5",
+    "bit3/contao-dependency-container": "<1.8"
```

Comment:
@discordier If replace as self.version, the package not install as symlink. What do you mean?